README.md

```diff
@@ -52,6 +52,7 @@ Please add yourself the first time you contribute.
 * Lane Biocini
 * Marc Bezem
 * Martin Escardo
+* Michael Shulman (iii)
```

Comment:
Thanks for the clarification... These results are unpublished joint work with Mike (but the mathematics are due to him). I'll ask him what he prefers.

Reply:
(ah, the thing I didn't mention is --- these results are not in his paper on idempotents.)

Reply:
It doesn't matter much to me.  I'm okay with being credited as an "indirect contributor" if that's a thing in this library.  On the other hand, since the joint paper with Jon will (fingers crossed) eventually be published, in the long run it would also make sense to just say that Jon did the contributions to the library and cite the paper as the source of the mathematics, even if the contributions to the library happened temporally before the paper was published.

Reply:
I'll let @jonsterling and @mikeshulman decide. When you are done, please let me know.

Reply:
Thanks @mikeshulman ! I think we can leave Mike as an indirect contributor, but once our paper is available let's add a citation to it to the appropriate files.

Reply:
How about if @mikeshulman gives you a preliminary title and list of authors, and you cite it as "in preparation", @jonsterling ?